src/utils/flags/consts.ts

```diff
@@ -0,0 +1,2 @@
+export const FLAG_NET_ATTACH_DEF = 'NET_ATTACH_DEF';
+export const FLAG_KUBEVIRT_DEF = 'KUBEVIRT';
```

Comment:
Here I'd just use `FLAG_KUBEVIRT`. IMHO no need to add "DEF" at the end of the name of this constant. Also it would be more consistent with the name of the other constant above.